week2/README.md

```diff
@@ -20,7 +22,7 @@ Last week we learned the basic building blocks of React and how to think in comp
 
 Firstly, it is time to work with API's again. In vanilla JavaScript we learned about the fetch mechanism, but when thinking in a component way we need to approach it a little differently. Take a look at the study book [here](https://study.hackyourfuture.net/#/react/api-calls.md) to learn how you can interact with an API using React.
 
-Secondly, it is time to start the dive into testing in React. Have a look at the `Paradigms`, `Testing-library` and `Mocking` sections of the study book [here](https://study.hackyourfuture.net/#/react/testing.md). `Enzyme` and `Working with time` are things that you may encounter or need in the future, so keep that in the back of your mind but those sections are optional if you have time. This is a lot of information and we will keep practicing with it next week. For now it is important to know about the `react-testing-library` and the way of thinking about testing: User centric!
+Next, let's look at routing. Because we build single page applications in React, the location in the address bar has no effect on what we should show. When building more advanced applications that becomes a problem, as we want to be able to define different pages. To solve this we use what is called client-side routing, have a look [here](https://study.hackyourfuture.net/#/react/routing.md).
```

Comment:
I believe this is a much more effective place to teach routing as well, reading through the curriculum, because it allows us to continue exploring what an SPA actually is, instead of as an afterthought.